app/helpers/external_report_export_helper.rb

```diff
@@ -0,0 +1,16 @@
+module ExternalReportExportHelper
```

Comment:
'MonthlyStatisticsExportHelper`?

Reply:
Yea have done. I'll. update the naming in the other exports as well as part of this PR